app/controllers/workshops_controller.rb

```diff
@@ -11,13 +11,18 @@ def render_workshop
     begin
       workshop = params[:workshop]
       path = workshops_path.join(workshop, 'README.md')
+      url = request.url
 
       @title = workshop.humanize.titleize
 
       @metadata_page_name = @title
       @metadata_page_category = 'Workshop'
+      @metadata_do_not_track = signed_in?
 
       render_md_file(path)
+
+      analytics.track_workshop_view(workshop, url) if current_user
```

Comment:
This can be combined with `@metadata_do_not_track` to simplify the page logic:

```ruby
if signed_in?
  analytics.track_workshop_view(workshop, url)
  @metadata_do_not_track = true
```

This change also has the positive side effect of not sending `@metadata_do_not_track` to the frontend at all when it's set to false.